### DISCUSSION TITLE
roboshop project

### DISCUSSION THREAD

**Discussion by GCKeita** [learndevopsonline/learndevopsonline#1459] [Q&A] [answered]
I am having this error:

[ centos@payment ~/roboshop-shell ]$ sudo bash payment.sh
roboshop_app_password is missing

can anyone help? thank you

**Comment by sreedharm07** [comment, marked as answer]
Hi @GCKeita 
i runned your repository  in my machine its executed fine, for your reference i have attached snap below.
i executed your code in my machine because you dint given your machine ip address
error cause may be you given wrong configration while creating instance. 
Delete the instance and create again with proper steps ,steps while creating = centos 8 ->proceed without key pair ->attach allow all security group
 even if you get same error please mention your ip address 


![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/3389cd78-57a2-4d97-b84e-79a7dec7fbbe)

**Reply by GCKeita** [reply]
thank you
